### DISCUSSION TITLE
如何在Form表单中给Upload组件回填默认值？

### DISCUSSION THREAD

**Discussion by taofeifanIT** [ant-design/ant-design#42930] [Q&A]
![image](https://github.com/ant-design/ant-design/assets/42193371/e2f4927e-1dff-4588-90c3-16362873cf1b)
![image](https://github.com/ant-design/ant-design/assets/42193371/89eab48e-9cc5-4bc6-90db-f09a4569a49a)

**Comment by Wxh16144** [comment]
可以帮助提供一个在线示例吗?
